examples/tv-app/linux/include/content-launcher/ContentLauncherManager.h

```diff
@@ -51,11 +51,15 @@ class ContentLauncherManager : public ContentLauncherDelegate
     CHIP_ERROR HandleGetAcceptHeaderList(AttributeValueEncoder & aEncoder) override;
     uint32_t HandleGetSupportedStreamingProtocols() override;
 
+    uint32_t GetFeatureMap(chip::EndpointId endpoint) override;
```

Comment:
Wont we have to do this for android platform as well?

Reply:
yes, that will be added to this PR